components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/java/org/wso2/carbon/identity/api/server/idp/v1/core/ServerIdpManagementService.java

```diff
@@ -2200,6 +2202,13 @@ private JustInTimeProvisioning createJITResponse(IdentityProvider idp) {
                 jitConfig.setUserstore(idp.getJustInTimeProvisioningConfig().getProvisioningUserStore());
             }
             jitConfig.setAssociateLocalUser(idp.getJustInTimeProvisioningConfig().isAssociateLocalUserEnabled());
+            if (idp.getJustInTimeProvisioningConfig().getSyncAttributeMethod() == null) {
```

Comment:
What if the value is empty or blank ?

Reply:
Since this is a ENUM, either it can be null or one of the values it supports. So  we can consider that it cannot be empty or blank